projectile-rails.el

```diff
@@ -207,6 +207,21 @@
 (defcustom projectile-rails-discover-bind "s-r"
   "The :bind option that will be passed `discover-add-context-menu' if available")
 
+(defcustom projectile-rails-rails-command "bundle exec rails"
```

Comment:
Thanks for the suggestion! I like that better than `rails-rails`, which reads a bit weird.